src/types/tweak_db_id.rs

```diff
@@ -64,6 +64,18 @@ impl TweakDbId {
             [b1, b2, b3],
         )
     }
+
+    pub(super) const fn hash(&self) -> u32 {
+        unsafe { self.0.__bindgen_anon_1.name }.hash
+    }
+
+    pub(super) const fn len(&self) -> u8 {
+        unsafe { self.0.__bindgen_anon_1.name }.length
+    }
+
+    pub(super) const fn into_inner(self) -> red::TweakDBID {
+        self.0
+    }
```

Comment:
`Copy` types shouldn't use `into_` in methos names according to [Rust API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv)
```suggestion
    pub(super) const fn to_inner(self) -> red::TweakDBID {
        self.0
    }
```

Reply:
renamed in 0ec8365